### PR TITLE
refactors

### PR DESCRIPTION
**Pull request by itsmebipul00**
Minute refactors

### CODE REVIEW

**Review by zoyatanweer — APPROVED** [review]
- code is clean after refactors.
- good to merge

**Review by tonymike99 — APPROVED** [review]
Hi Bipul,
The code looks fine. Good to merge.

File: src/ActionProviders/VideoActions.js [hunk 1]
```diff
@@ -68,6 +68,8 @@ const VideosProvider = props => {
 			payload: data,
 		})
 
+	// const implementSetTimeout = (cb) => setTimeout(() => cb, 1000)
```
Comment (sudipkundu999) [inline src/ActionProviders/VideoActions.js:71]: Remove commented code

**Review by sudipkundu999 — APPROVED** [review]
Good work. Good to merge

**Pull request merged**